<launch>
  <node pkg = "suture_knot" type = "suture_knot_client" name="knot_client"/>
</launch>
